x-pack/plugins/ml/public/application/util/string_utils.ts

```diff
@@ -129,6 +130,10 @@ export function escapeForElasticsearchQuery(str: string): string {
   return String(str).replace(/[-[\]{}()+!<>=?:\/\\^"~*&|\s]/g, '\\$&');
 }
 
+export function escapeKueryForFieldValuePair(name: any, value: any): string {
```

Comment:
we should avoid using `any`
```suggestion
export function escapeKueryForFieldValuePair(name: string, value: unknown): string {
```

Reply:
Updated here [`5259647` (#140147)](https://github.com/elastic/kibana/pull/140147/commits/5259647685d29c05657c2b8f57e0b2ae4e68e2e4)